appendix-basic65.tex

```diff
@@ -804,9 +804,9 @@ \subsection{BORDER}
 \item [Usage:] Sets the border colour
                of the screen to the argument, which must be in the
                range of 0 to 255. All colours within this range are
-               customisable via the {\bf PALETTE} command, though on
-               startup, the ROM provides the first 32 preset colours
-               described in the colour table under {\bf BACKGROUND}
+               customisable via the {\bf PALETTE} command. On
+               startup, the MEGA65 has only the first 32 colours configured,
+               which are described in the table under {\bf BACKGROUND}
```

Comment:
under the **BACKGROUND** command on page X

Reply:
done